build.gradle

```diff
@@ -253,6 +253,9 @@ allprojects {
       // ignore PatternMatchingInstanceof for now
       check("PatternMatchingInstanceof", net.ltgt.gradle.errorprone.CheckSeverity.OFF)
 
+      // ignore EffectivelyPrivate for now
+      check("EffectivelyPrivate", net.ltgt.gradle.errorprone.CheckSeverity.OFF)
```

Comment:
Decided to disable this check for now. We can revisit this idea if we feel like it is a good check.